### ISSUE TITLE
[QUESTION] Pgreg / pattern/ exe roadmap status

### ISSUE THREAD

**Issue by discoduck2x** [Staples-Inc/snap-plugin-collector-procstat#1]
Is there any timeframe for getting the pgrep func into this plugin? either i have go go through making everything currently not creating pid files do that or get this via pgrep pattern etc

**Comment by jtlisi** [comment]
@discoduck2x There is no current time frame on this feature. I can try to have a working PR ready by next week at some point if that works for you?

**Comment by discoduck2x** [comment]
oh @jtlisi sounds great - if its something i can help out with to test etc just let me know - unfort i dont code so cant assist there :)

**Comment by jtlisi** [comment]
Of course, I can add it to my backlog for work this week. I'll have a PR by Friday.
